app/policies/recipe_policy.rb

```diff
@@ -0,0 +1,53 @@
+class RecipePolicy < ApplicationPolicy
+  attr_reader :user, :recipe
+
+  def initialize(user, recipe)
+    @user = user
+    @recipe = recipe
+  end
```

Comment:
You can just remove these, and then update the `owner?` method to use `record` instead of `recipe`. You might even want to consider moving the `owner?` method to the `ApplicationPolicy` becasue it would be useful for other policies in the future.